components/component-stylesheets/nav.css

```diff
@@ -0,0 +1,61 @@
+.nav{
+    display: flex;
+    width: 100%;
+    column-gap: 50px;
```

Comment:
you can use rem unit for better resonsive

Reply:
Thanks for pointing out, fixed in recent commit